README.md

```diff
@@ -1,6 +1,21 @@
 # my-amethyst-project
 game development using the Amethyst game engine in Rust
 
+## Purpose
+Create a focused, opinionated fighting game engine from Amethyst.
+
+Initial support goals for the project are.
+
+- Rollback
```

Comment:
More like, serializable game state is the thing that opens up the door to be able to do Rollback netcode (where "predict user inputs" is just a detail of that algorithm) as well as being able to rewind in general

Reply:
Or like, we need to have a serializable, rewindable game state, and then the rollback feature is just one of the things enabled by that

Reply:
so remove lines 10 and 12, and just have it says Serializeable?

Reply:
I think just change the order so Serializable game state is at the top and then supporting rollback netcode is one of the bullet points under it

Reply:
and remove 'predict user inputs'